webhook/webhook.groovy

```diff
@@ -512,8 +512,8 @@ class WebHook {
                 for (def webhookListener : webhookListeners) {
                     try {
                         if (!webhookListener.isAsync())
-                            if (eventPassedFilters(event, json, webhook))
```

Comment:
note: no 'webhook' variable in scope here, should likely be 'webhookListener' to be correct